Makefile

```diff
@@ -17,6 +18,25 @@ release:
 		aws s3 cp build/$$os/layer0_$(L0_VERSION)_$$os.zip s3://xfra-layer0/release/$(L0_VERSION)/layer0_$(L0_VERSION)_$$os.zip ; \
 	done
 
+github-release:
+	$(MAKE) -C cli release
+	$(MAKE) -C setup release
+	$(MAKE) -C plugins/terraform release
+
+	rm -rf binaries
+	cp -R cli/build/ binaries 
+	cp -R setup/build/ binaries
+	cp -R plugins/terraform/build/  binaries 
+
+	zip -r binaries.zip binaries
+
+update-release:
```

Comment:
Updating the docs' version references should be handled in a `release` target in `layer0/docs-src/Makefile`.

For example, in `layer0/Makefile`:
```
release:
        $(MAKE) -C api release
        $(MAKE) -C cli release
        $(MAKE) -C docs release                     <- added here
        $(MAKE) -C runner release
        $(MAKE) -C setup release
        $(MAKE) -C plugins/terraform release

        rm -rf build
        ...
```

and in `layer0/docs-src/Makefile`
```
build:
        mkdocs build --clean

release:
    # sed commands ...
    $(MAKE) build
```